core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -176,3 +191,8 @@ final class ParallelLeftSequenceOps[T[_, _], M[_], A, B](private val tmab: T[M[A
   def parLeftSequence(implicit T: Bitraverse[T], P: Parallel[M]): M[T[A, B]] =
     Parallel.parLeftSequence(tmab)
 }
+
+final class ParallelFoldMapAOps[T[_], A](private val ma: T[A]) extends AnyVal {
```

Comment:
It's really unfortunate what we've done here (not you—but us, the maintainers). `ParallelTraversableOps`, `ParallelTraversable_Ops`, `ParallelFlatTraversableOps`, `ParallelFoldMapAOps`, etc. could all be a single `ParallelFoldableOps` trait. As it is I guess we should go with what you've got here for consistency.